tests/config/test_data.config

```diff
@@ -723,6 +723,10 @@ params {
                 dbsnp_138_b37_chr17_vcf_gz                     = "${params.test_data_base_msk}/hg37/genomic_reference/GRCH37/vcf/dbsnp_138.b37.chr17.vcf.gz"
                 dbsnp_138_b37_chr17_vcf_gz_tbi                 = "${params.test_data_base_msk}/hg37/genomic_reference/GRCH37/vcf/dbsnp_138.b37.chr17.vcf.gz.tbi"
             }
+            'chr22' {
+                chr22maf                 = "${params.test_data_base_msk}/chr22/chr22/chr22.maf"
```

Comment:
fairly minor but can you align the ='s like in the rest of the file

Reply:
fixed!